Dockerfile

```diff
@@ -43,6 +43,12 @@ RUN apt-get update \
 		wget \
 	&& rm -rf /var/lib/apt/lists/*
 
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ buster-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
```

Comment:
If we're going to add this package, I'd personally rather it was added in the list of all packages in the `RUN` above (and if the sources require adding, doing this as the first line), rather than having this as a separate set of `apt-get` install. It doesn't make much sense for it to be separate.

Reply:
Actually, sorry, this might be a silly question. Why's this being added here as opposed to downstream in (I assume) the Jellyfish service that pulls this in as it's parent? Almost all of the children this base image is a parent for won't have use for this, whilst several have use of the other utils we pull in (though looking through the list, I'm starting to wonder if some of these should be extracted, too...).